tests/test_spectra_matching.py

```diff
@@ -127,27 +126,67 @@ def test_create_spectra_generator_spectrum_source(small_mgf_path):
     assert isinstance(spectra_generator.__next__(), Spectrum)
 
 
-def test_make_batch_requests(small_mgf_path):
-    client = SpectraMatching()
-    _46_spectra = list(load_from_mgf(small_mgf_path))
+@pytest.fixture
+def spectra_matching_client(mock_send_request):
+    client = SpectraMatching(optional_token="token")
```

Comment:
don't need to pass token here I think

Reply:
If I don't, this function is executed `authenticate_client`. I wanted to avoid it